cfep-03.md

```diff
@@ -0,0 +1,72 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (CircleCI, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
+
+An example is the Qt5 build that seems to be very hard to build (it tooks almost a year for Continuum folks to compile it for all platforms). See the Qt5 recipe build discussion ticket here https://github.com/conda-forge/qt-feedstock/issues/5.
+
+That being said it's excepted that a small amount of packages will be very hard to automatically build for various reasons. 
+
+Obviously we all want to keep the amount of packages small and have a maximum of automatically build packages on conda-forge.
+
+But a system/protocol to manually upload builds on conda-froge anaconda channel is necessary to circumvent some situations.
+
+## Specification
+
+A protocol to manually upload builds could look something like that :
+
+- The user provides the recipe as a PR.
+
+- The build fails : someone from @conda-forge/core decides to "enable" the manually uploaded builds system.
+
+- People from @conda-forge/core are designated to build, test and upload the package.
+
+For Linux, the `scripts/run_docker_build.sh` should be used (builds are placed in `build_artefacts/`. Building with this script will provide consistency for these packages because they will be build against the same Linux distribution.
+
+On Windows, we could use this : ContinuumIO/vagrant-images#1. (Need help here I have no expertise with Windows)
```

Comment:
We would likely need to ensure we have our own vagrant image with all the correct compilers set up - but certainly vagrant is our next closest thing to docker.

Reply:
Indeed. @mariusvniekerk created a VM to build `scipy` and we should document the steps he took to achieve that. However, that should be in the docs and not in the cfep.